Signal/src/AppDelegate.m

```diff
@@ -63,6 +63,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [logger addLoggingCallback:^(NSString *category, id details, NSUInteger index){
     }];
 
+    // XXX - careful when moving this. It must happen before we initialize TSStorageManager.
+    [self verifyDBKeysAvailableBeforeBackgroundLaunch];
```

Comment:
I'd really like to review our ordering of operations in app startup. It looks like this is being done _before_ logging is enabled down below which is probably a serious issue.